packages/vue/src/components/FCrudDataset/FCrudButton.vue

```diff
@@ -75,14 +75,11 @@ function executeAction(): void {
 </script>
 
 <template>
-    <button type="button" class="button button--small button--tertiary" @click="executeAction">
-        <f-icon v-if="iconName" class="button__icon" :name="iconName"></f-icon>
-        <span v-if="!props.label" class="sr-only">
-            <!--
-                 @slot Slot used to provide custom content for the button text.
-            -->
-            <slot> {{ buttonText }} </slot>
-        </span>
-        <slot v-if="props.label"> {{ buttonText }} </slot>
-    </button>
+    <!-- [html-validate-disable-block element-permitted-content, element-required-ancestor -- this is now the responsibility of FCrudButton instead] -->
+    <f-table-button :icon="iconName" :label="props.label" @click="executeAction">
```

Comment:
Kom på att detta kanske inte är så bra ändå eftersom man kan lägga in en `FList` i crudden. Det fungerar och det ser rätt ut men klassen blir `table__button` vilket kanske inte är så snyggt?

Vi skulle kunna upptäcka om man specifikt är i tabell (inject table mode typ?) och bara använda `FTableButton` då och annars fallback på vanlig knapp med klasserna ovan.

Alternativt så struntar vi i det och kastar ur `FCrudButton` helt och då får konsumenten ta hand om det genom att använda `FTableButton` om man stoppat in den i en tabell annars vilken typ av knapp man nu vill ha. Ändringen i `FCrudDataset` möjliggör ju att man själv stoppar in `<button>` ändå.

Reply:
Vi låter det vara, uppdaterat alla exempel att inte använda `FCrudButton` och lägger en PR senare för att deprektera komponenten.